src/dfmpreview/.gitignore

```diff
@@ -0,0 +1,34 @@
+###############
+#    folder   #
+###############
+node_modules/
+out/
+
+
+###############
+#  temp file  #
+###############
+*.nupkg
+*.log
+*.tmp
+*.vsprops
+*.suo
+*.user
+*.userprefs
+*.sdf
+*.sln.ide
+*.cache
+*.sln.metaproj
+.gitattributes
+.settings
+log.txt
+debugSettings.json
+launchSettings.json
+project.lock.json
+*.sublime-workspace
+*.nuget.targets
+
+###############
+#  template   #
+###############
+src/docfx/Template/*.zip
```

Comment:
I forgot to remove the useless lines ,but the folder 'out' and 'node_modules' should not be pushed,should I write it in '.gitignore' which in the root directory

---
In reply to: [73490158](https://github.com/dotnet/docfx/pull/558#discussion_r73490158) [](ancestors = 73490158,73489606)